src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+
+type Props = {
+  errorMessage: string,
+  setErrorMessage: (param: string) => void,
+};
+
+export const ErrorMessage: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  const [hiddenError, setHiddenError] = useState(false);
+
+  if (errorMessage) {
+    setTimeout(() => setErrorMessage(''), 3000);
+  }
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        {
+          hidden: hiddenError,
+        },
+      )}
+
+    >
+      <button
+        aria-label="a problem"
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => setHiddenError(prev => !prev)}
```

Comment:
You can make a separate function above return  `  const handleErrorMessage = () => {
    setHiddenError(prev => !prev);
  };`  and then put here only a name of the function